### DISCUSSION TITLE
2.5 to 2.6

### DISCUSSION THREAD

**Discussion by Mikey1968** [EdgeTX/edgetx#1783] [Q&A]
Im trying to go from 2.5 to 2.6 but when I do my models are gone, people say they have to be converted . People have told me different ways to do it and doesnt work.  I wish someone would make a video on how its done 
Anybody out there can help me step by step

**Comment by pfeerick** [comment]
If you have migrated from EdgeTX 2.5 -> 2.6, and for some reason the models haven't been converted over on the radio, you should be able to do this in Compaion. First of all, what radio is this with? ;) As B&W radios are different to colorlcd radios. 

But basically, you would run Companion 2.5, read the old models from the radio (when the radio is in bootloader mode), save it to an otx file. 
You would then run Companion 2.6, read that otx file, and write it to the radio (again, when in bootloader mode).

If you have a B&W radio, you need to be running the 2.5 bootloader to read the old models, so that the EEPROM (where the settings were stored pre 2.6) is accessible. After that though, you can run the bootloader that matches the firmware version you are using.

**Comment by rotorman** [comment]
@Mikey1968 I answered a similar post in RCGroups: https://www.rcgroups.com/forums/showthread.php?3916381-Official-EdgeTX-Discussion-Thread/page157#post48953385

Please see a video by John/RC Video Reviews explicitly about the topic of binary to YAML conversion: https://www.youtube.com/watch?v=18KKJxU2UH0
